docs/HowToDevelopThingsUsingSTDK.md

```diff
@@ -0,0 +1,256 @@
+# How to develop things using STDK based TizenRT   
+
+STDK is one of Develop Kit(STDK) for Smart Things Eco-system.   
+This guide shows an example how to enable STDK and develop things.   
+
```

Comment:
You could rewrite as "STDK (Smart Things Develop Kit) is one of the features of the Smart Things ecosystem. This guide describes with an example, how to enable STDK in TizenRT and develop applications using it."

Reply:
Has Changed.